packages/flutter_tools/lib/src/ios/code_signing.dart

```diff
@@ -116,6 +118,53 @@ Future<Map<String, String>?> getCodeSigningIdentityDevelopmentTeam({
     return null;
   }
 
+  final String? developmentTeam = await _getCodeSigningIdentityDevelopmentTeam(
+    processManager: processManager,
+    platform: platform,
+    logger: logger,
+    config: config,
+    terminal: terminal,
+    shouldExitOnNoCerts: true,
+  );
+
+  if (developmentTeam == null) {
+    return null;
+  }
+
+  return <String, String>{
+    'DEVELOPMENT_TEAM': developmentTeam,
```

Comment:
nits: Have 'DEVELOPMENT_TEAM' assigned to a constant and use the constant instead.